js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -367,6 +367,14 @@ const ValidatingController = modules.Controller.inherit((function() {
                 if(showEditorAlways && editingController.isCellOrBatchEditMode() && editingController.allowUpdating({ row: parameters.row })) {
                     editIndex = editingController._addEditData({ key: parameters.key, oldData: parameters.data });
                 }
+
+                if(this.option('editing.editRowKey') === parameters.key) {
+                    editIndex = 0;
```

Comment:
Why?

Reply:
Because changes are empty on editing start => editIndex is -1 => validators will not be created